addons/web/static/src/scss/bootstrap_review_backend.scss

```diff
@@ -253,3 +253,18 @@ $-o-bg-colors-custom: null;
     background-color: $input-focus-bg;
     border-color: $o-input-hover-border-color;
 }
+
+
+// Inputs fields
+// Creating a different styling for toggle switches with a purpose of switching from A to B rather than ON/OFF
```

Comment:
columns of max. 80 characters wide 😉
https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#syntax-and-formatting